bin/migrate-oats-data/prod_db_obfuscation/alcs/inquiry/inquiry.py

```diff
@@ -0,0 +1,238 @@
+from common import setup_and_get_logger
+from db import inject_conn_pool
+from ...utils import log_process, get_update_column_query
+from faker import Faker
+
+logger = setup_and_get_logger("prod_data_obfuscation")
+
+
+def obfuscate_inquiry_related_data():
+    _update_inquiries_summary()
+    _update_inquiries_first_name()
+    _update_inquiries_last_name()
+    _update_inquiries_organization()
+    _update_inquiries_email()
+    _update_inquiries_phone()
+    _update_inquiry_parcel()
+
+
+@log_process(logger, "update_with_random_alcs_inquiries")
+@inject_conn_pool
+def _update_inquiries_summary(conn=None):
+    fake = Faker()
+    fake_latin = Faker("la")
+
+    try:
+        with conn.cursor() as cursor:
+            cursor.execute("SELECT uuid FROM alcs.inquiry")
+            record_ids = cursor.fetchall()
+
+            for r_id in record_ids:
+                sql_query = f"""
+                            UPDATE alcs.inquiry 
+                            SET 
+                            {get_update_column_query("summary")}
+                            WHERE uuid = %s AND summary IS NOT NULL;
```

Comment:
I can see that you've opted for splitting the update into multiple queries, but for the future, you could also look into this: "column1 = CASE WHEN column1 IS NOT NULL THEN new_value1 ELSE column1 END," It will achieve the same result and a bit easier to read.